src/printer.js

```diff
@@ -137,7 +136,7 @@ function genericPrint(path, options, printPath, args) {
           decorator.type === "TSDecorator"
           ? ""
           : "@";
-        parts.push(prefix, printPath(decoratorPath), line);
+        decorators.push(prefix, printPath(decoratorPath), hardline);
```

Comment:
line used to work by accident, because we didn't have any group so the other properties would likely make it break. We always do want to break here.